From mmarek@suse.cz  Tue Jun  2 12:28:08 2009
From: Michal Marek <mmarek@suse.cz>
Date: Tue, 26 May 2009 14:19:35 +0200
Subject: kobject: samples: make SAMPLE_KOBJECT module-only
To: Greg Kroah-Hartman <gregkh@suse.de>
Message-ID: <20090526121935.GA20790@sepie.suse.cz>


With SAMPLE_KOBJECT=y, it isn't even linked into the kernel image.

Signed-off-by: Michal Marek <mmarek@suse.cz>
Signed-off-by: Greg Kroah-Hartman <gregkh@suse.de>

---
 samples/Kconfig |    3 ++-
 1 file changed, 2 insertions(+), 1 deletion(-)

--- a/samples/Kconfig
+++ b/samples/Kconfig
@@ -20,7 +20,8 @@ config SAMPLE_TRACEPOINTS
 	  This build tracepoints example modules.
 
 config SAMPLE_KOBJECT
-	tristate "Build kobject examples"
+	tristate "Build kobject examples -- loadable modules only"
+	depends on m
 	help
 	  This config option will allow you to build a number of
 	  different kobject sample modules showing how to use kobjects,
